packages/web-components/src/components/va-radio/va-radio.tsx

```diff
@@ -220,24 +220,24 @@ export class VaRadio {
         'usa-legend': true,
         'usa-label--error': error
       });
-      return ( 
+      return (
         <Host
           aria-invalid={error ? 'true' : 'false'}
           aria-label={ariaLabel}
         >
           <fieldset class="usa-form usa-fieldset">
-            <legend class={legendClass}>
+            <legend class={legendClass} part="legend">
               {label}
-              {required && <span class="usa-label--required"> {i18next.t('required')}</span>}
+              {required && <span class="usa-label--required" part="required"> {i18next.t('required')}</span>}
             </legend>
-            <span class="usa-error-message" role="alert">
+            <span class="usa-error-message" role="alert" part="error">
```

Comment:
Is there a specific scenario that you need to be able to override error style?

Reply:
I don't have one, but added this if we need it in the future... I can remove it if you don't think it's needed